tests/Unit/SensuConfigServiceTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Tests\Unit;
+
+use PHPUnit\Framework\TestCase;
+use SensuDashboard\Exception\SensorConfigurationNotSetException;
+use SensuDashboard\Service\SensuConfigService;
+use UnexpectedValueException;
+
+class SensuConfigServiceTest extends TestCase
+{
+    public function setUp(): void
+    {
+        mkDir(__DIR__ . '/sensor-config');
+        mkDir(__DIR__ . '/sensor-config/iaptus-uk-test');
+        mkDir(__DIR__ . '/sensor-config/iaptus-aus-test');
+
+        $file = fopen(__DIR__ . '/sensor-config/iaptus-uk-test/test-sensor.json', 'w+');
+        fwrite($file, json_encode(['sensor-uk' => 'green']));
+        fclose($file);
+
+        $file = fopen(__DIR__ . '/sensor-config/iaptus-aus-test/test-sensor.json', 'w+');
+        fwrite($file, json_encode(['sensor-ca' => 'red']));
```

Comment:
Is this supposed to be `ca` not `aus`?

Reply:
Yes :( a86d62f2d495719321ce5ae43dbb9c53be4f5e95